app/views/welcome/index.html.erb

```diff
@@ -17,41 +17,38 @@
   <div class="large-4 columns">
     <img src="http://placehold.it/400x300&text=[img]"/>
     <h4>Spend wisely.</h4>
-    <p>Bacon ipsum dolor sit amet nulla ham qui sint exercitation eiusmod commodo, chuck duis velit. Aute in reprehenderit, dolore aliqua non est magna in labore pig pork biltong. Eiusmod swine spare ribs reprehenderit culpa. Boudin aliqua adipisicing rump corned beef.</p>
+    <p>Knowing when to spend is important when managing your money. Keep track of your expenses and savings so you know when the right time is to make a big purchase. Looking to buy something special? We'll help you prepare!</p>
   </div>
 
   <div class="large-4 columns">
     <img src="http://placehold.it/400x300&text=[img]"/>
     <h4>Save money.</h4>
-    <p>Bacon ipsum dolor sit amet nulla ham qui sint exercitation eiusmod commodo, chuck duis velit. Aute in reprehenderit, dolore aliqua non est magna in labore pig pork biltong. Eiusmod swine spare ribs reprehenderit culpa. Boudin aliqua adipisicing rump corned beef.</p>
+    <p>Keep track of your savings so you are prepared for a rainy day or you're looking to treat yourself! Build a plan to save now so you won't worry later.</p>
   </div>
 
   <div class="large-4 columns">
     <img src="http://placehold.it/400x300&text=[img]"/>
     <h4>Compare spending habits.</h4>
-    <p>Bacon ipsum dolor sit amet nulla ham qui sint exercitation eiusmod commodo, chuck duis velit. Aute in reprehenderit, dolore aliqua non est magna in labore pig pork biltong. Eiusmod swine spare ribs reprehenderit culpa. Boudin aliqua adipisicing rump corned beef.</p>
-  </div>
-
+    <p>Determine if how much you are spending each month will set you up for successful saving. Compare your spending habits to those of others and learn how to maximize your savings!</p>
   </div>
+</div>
 
 
 <div class="row">
-  <div class="large-12 columns">
-
-    <div class="panel">
-      <h4>Get in touch!</h4>
+  <h4>About the creators</h4>
```

Comment:
feels weird that our about us divs are larger than the 3 value prop divs, we should refactor this in the future so that either we are smaller, or we can move this into it's own about us page.  i'm good with how this looks for now